cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -453,6 +459,88 @@ func (dc *DownstreamController) syncEndpoints() {
 	}
 }
 
+func (dc *DownstreamController) syncRule() {
+	for {
+		select {
+		case <-beehiveContext.Done():
+			klog.Warning("Stop edgecontroller downstream syncRule loop")
+			return
+		case e := <-dc.rulesManager.Events():
+			klog.V(4).Infof("Get rule events: event type: %s.", e.Type)
+			rule, ok := e.Object.(*routerv1.Rule)
+			if !ok {
+				klog.Warningf("object type: %T unsupported", rule)
```

Comment:
if not !ok, then rule is empty, print rule is invalid. should use `e`?

Reply:
it will continue next loop to get next event

Reply:
sorry, i misunderstand it, and i have fix it